eng/pipelines/official.yml

```diff
@@ -107,12 +107,37 @@ extends:
       - template: /eng/pipelines/templates/jobs/workload-build.yml@self
         parameters:
           stabilizePackageVersion: ${{ parameters.stabilizePackageVersion }}
-          publishToFeed: ${{ parameters.publishToFeed }}
-          feedForPublishing: ${{ parameters.feedForPublishing }}
           createVSInsertion: ${{ parameters.createVSInsertion }}
           usePreComponentsForVSInsertion: ${{ parameters.usePreComponentsForVSInsertion }}
           includeNonShippingWorkloads: ${{ parameters.includeNonShippingWorkloads }}
           vsTopicBranch: ${{ parameters.vsTopicBranch }}
           workloadDropNames: ${{ parameters.workloadDropNames }}
           primaryVsInsertionBranches: ${{ parameters.primaryVsInsertionBranches }}
           secondaryVsInsertionBranches: ${{ parameters.secondaryVsInsertionBranches }}
+    - stage: Publish to feed
+      displayName: Publish
+      dependsOn: Build
+      condition: succeeded()
+      jobs:
+      - deployment: Publish
+        environment: DotNet-SDK-Workloads
+        pool:
+          name: $(DncEngInternalBuildPool)
+          image: 1es-windows-2022
+          os: windows
+        templateContext:
+          type: releaseJob
+          isProduction: true
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+              - task: 1ES.DownloadPipelineArtifact@1
+                displayName: 📥 Download PackageArtifacts
```

Comment:
I used purple circles `🟣` to represent a step that runs in this repo. To keep consistent, probably use that instead of `📥`.